turbopack/crates/turbopack-ecmascript/src/tree_shake/asset.rs

```diff
@@ -100,6 +100,7 @@ impl EcmascriptModulePartAsset {
             ModulePart::Internal(..)
                 | ModulePart::InternalEvaluation(..)
                 | ModulePart::Facade
+                | ModulePart::Export(..)
```

Comment:
This is a fix for duplicate modules. cc @mischnic 

Previously, an export fragment was resolved as an internal fragment, which causes duplicate module identifier